Fix issue 47 (plot with no finite points)

dlm opened the pull request:
Proposed solution for issue https://github.com/mrzv/dionysus/issues/47

tests/test_issue47.py gives a small example of the issue.
Solution defines the default plotting region as [(0,0), (1,1)],

mrzv commented:
@dlm Thanks. This is great. A couple of minor things.

1. The code in `test_issue47.py` needs to be in a function, e.g., `test_plot`. Otherwise, `pytest` doesn't see it.
2. I wonder if `min_/max_death` should default to `min_/max_birth` rather than 0,1?
3. The comment in `test_issue47.py` about how the error gets triggered doesn't make sense once it's fixed. Let's just drop it.

Thanks again!

dlm commented:
No problem and thanks for the great feedback!
1. addressed
2. good call!  I updated
3. sounds good.  comments removed.

Please, let me know if there are other updates.

mrzv commented:
Perfect. Thanks, Dave.
